### ISSUE TITLE
Docs needed to explain how to use IoTHub Trigger

### ISSUE THREAD

**Issue by wickste** [Azure/Azure-Functions#986]
We expose IoTHub Trigger as a concept in VS2017, CLI and VSCode, however searching for it on docs.microsoft.com yields zero results. There is no tutorial, reference topic or any mention at all. Having docs on this is critical because it is not obvious how to configure it correctly with the connection strings - especially in the light of the breaking change between V1 and V2 here.

We should strive to:
1) have a dedicated topic similar to this one https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-event-hubs 
2) mention it here like all the other triggers: https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview 
3) mention it here https://docs.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings
4) mention it here https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local#create-func
5) mention it here https://docs.microsoft.com/en-us/azure/azure-functions/functions-develop-vs

**Comment by jeffhollan** [comment]
@ggailey777 know we had a thread on this. Just confirming we have this created in VSTS as well to track?

**Comment by jeffhollan** [comment]
I think this is still a valid ask. Going to assign to @ggailey777 and @asavaritayal for now to chat about how this should show up.  It is a very popular trigger that has no docs - even if the docs reference that it's powered by Event Hubs and what properties to expect.  Feel free to ping this issue once a DevOps item is created in docs board as that's likely best place for this ask to live.

**Comment by ggailey777** [comment]
[This work item](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1361771) is currently assigned to @ColbyTresness

**Comment by ColbyTresness** [comment]
Per the thread I started offline I don't think this should be assigned to me. Lets figure it out plans there and come back to this.

**Comment by ColbyTresness** [comment]
@craigshoemaker

**Comment by craigshoemaker** [comment]
@wickste, @ColbyTresness: [Azure IoT Hub bindings for Azure Functions](https://docs.microsoft.com/azure/azure-functions/functions-bindings-event-iot) is now available. Items 2- 5 are in progress.

**Comment by wickste** [comment]
@craigshoemaker @ColbyTresness For Functions version 1.x the user needs to provide the IoTHubConnectionString (not the event hub string)

**Comment by craigshoemaker** [comment]
@wickste just to clarify, are you saying that for all v1 examples each instance of `connection` should reference the IoTHubConnectionString and not the event hub connection string?

**Comment by wickste** [comment]
@craigshoemaker that's correct. If you use the event hub string with a V1 function it won't work. Have to use the IoTHub connection string in this case. This is an important callout to make very visible in the docs, as it's a common customer pitfall.
